/marker_publisher:
  ros__parameters:
    mesh_server_port: 8000
    mesh_resource_dir: ""  # must be set
    marker: None  # must be set
    marker_topic: "/visualization_marker"
    publish_interval: 1.0